src/components/list.js

```diff
@@ -3,6 +3,50 @@ import PropTypes from 'prop-types';
 import { getStyles } from '../utils/base';
 import styled from 'react-emotion';
 
+export const bulletStyles = {
+  star: '\\2605',
+  classicCheck: '\\2713',
+  greenCheck: '\\2705',
+  arrow: '\\219d',
+  cross: '\\274C'
+};
+
+export const getBulletStyle = (bulletStyle, isListItemStyle) => {
+  if (bulletStyle) {
+    const content = bulletStyles[bulletStyle] || `\\${bulletStyle}`;
+
+    if (isListItemStyle === true) {
+      return [
+        { listStyleType: 'none' },
+        `&::before {
+          content: '${content}' !important;
+          display: inline-block;
+          margin-right: 40px;
+          width: 20px;
+          font-size: 20px;
+          text-align: center;
+          vertical-align: middle;
+        }`
+      ];
+    } else {
+      return [
+        { listStyleType: 'none' },
+        `li::before {
+          content: '${content}';
+          display: inline-block;
+          margin-right: 40px;
+          width: 20px;
+          font-size: 20px;
+          text-align: center;
+          vertical-align: middle;
```

Comment:
Hmmm... it looks like these styles are shared by both the `li::before` and `&::before` style blocks:

```
display: inline-block;
margin-right: 40px;
width: 20px;
font-size: 20px;
text-align: center;
vertical-align: middle;
```

I feel there should be a way to share them so we have don't two separate style blocks to maintain should we need to update them styles 😎